### DISCUSSION TITLE
email said I need to reverify my academic status but I cannot finish

### DISCUSSION THREAD

**Discussion by xqe2011** [community/community#51111] [GitHub Education] [answered]
I am a member of GitHub Education. Yesterday, I have received an email which said I must reverify my academic status to keep the permission to access Education benefits. But when I followed the email, open to the **submit agin** page, and enter my school name, the page said I must have an education email. Our school doesn't provide email access to students expect for paper delivery. Can I upload a PDF from CHSI(an authority organization from china providing verification to academic status) instead?
It's very important for me because I am using a lots of education benefits. Hope for early reply, thank you!
My School is Dongguan University of Technology, and I am in my undergraduate.

**Comment by samus-aran** [comment]
Hi @xqe2011 👋 

Let's see how we can assist ⭐ 

What is the error message are you seeing on the application page?

**Reply by xqe2011** [reply]
<img width="603" alt="image" src="https://user-images.githubusercontent.com/16936819/229265513-25bee6a4-7904-4207-8928-b45d0dc2625f.png">

**Comment by sameerch4n** [comment]
Hello @xqe2011 
I understand that you are facing an issue accessing the GitHub Student Developer Pack, similar to what I encountered while attempting to register without a student email. It seems that you have not yet received a Student ID. In this case, I would recommend using a PDF of your Academic Status as proof that you are a part of the program. This approach worked for me and could work for you as well. I suggest seeking support from the technical department to assist you in obtaining a student subscription or account.

**Comment by martian0x80** [comment, marked as answer]
A workaround is to not select the suggested name for your uni in the drop down list when writing, so GitHub doesn't know the domain or website to your uni, this allows you to manually add the uni by selecting the address on the map on the next page, and does not require a registered email.
@xqe2011
